From 2fe230bb1be7be07867644dcb341e492b1ae9141 Mon Sep 17 00:00:00 2001
From: Floris Bos <bos@je-eigen-domein.nl>
Date: Fri, 4 Oct 2019 16:41:30 +0200
Subject: [PATCH 560/587] pcie-brcmstb-bounce64.c: dev_err() -> dev_info() for
 info messages

"dmabounce: initialised" is not an error, so do not log it as such.
Prevents screen polution on OS with "quiet" as kernel parameter.

Closes #3266
---
 drivers/pci/controller/pcie-brcmstb-bounce64.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/pci/controller/pcie-brcmstb-bounce64.c b/drivers/pci/controller/pcie-brcmstb-bounce64.c
index a8e7970c879d..9298d5da5023 100644
--- a/drivers/pci/controller/pcie-brcmstb-bounce64.c
+++ b/drivers/pci/controller/pcie-brcmstb-bounce64.c
@@ -516,7 +516,7 @@ int brcm_pcie_bounce_init(struct device *dev,
 
 	g_dmabounce_device_info = device_info;
 
-	dev_err(dev, "dmabounce: initialised - %ld kB, threshold %pad\n",
+	dev_info(dev, "dmabounce: initialised - %ld kB, threshold %pad\n",
 		 buffer_size / 1024, &threshold);
 
 	return 0;
-- 
2.17.1

